src/Sylius/Bundle/AdminBundle/templates/product/show/content/page_body/details/shared/shipping/width.html.twig

```diff
@@ -0,0 +1,12 @@
+{% set variant = hookable_metadata.context.variant %}
+
+<div class="row">
+    <div class="col-12 col-md-4 fw-bold">{{ 'sylius.ui.width'|trans }}:</div>
+    <div class="col-12 col-md-8">
+        {% if variant.width is null %}
+            <i>{{ 'sylius.ui.no_width'|trans }}</i>
```

Comment:
Why is this an icon?

Reply:
it's not the icon, it just makes the text italic